content/python/concepts/random-module/terms/randrange/randrange.md

```diff
@@ -13,29 +13,29 @@ CatalogContent:
   - 'paths/data-science'
 ---
 
-The `random.randrange()` method generates random numbers from a defined range of `int` values.
+The `random.randrange()` function generates random numbers from a defined range of `int` values.
```

Comment:
```suggestion
The `random.randrange()` function selects a random number from a defined range of `int` values.
```

Reply:
I'd link to range, but I don't think we have an entry for it yet.

Reply:
That was exactly my thought when I was looking at this file. 😄